src/file-box.ts

```diff
@@ -1017,14 +1017,12 @@ class FileBox implements Pipeable, FileBoxInterface {
   pipe<T extends Writable> (
     destination: T,
   ): T {
-    this.toStream().then(stream => {
-      stream.on('error', e => {
-        console.info('error:', e)
```

Comment:
Why do we remote the error information logging?

maybe change to `console.error` will be better.

Reply:
I think as a library there shouldn't be an output that can't be turned off externally, unless it's a coding error. But obviously the error here is not because of coding errors, so it should not be output, and should be passed out in the form of throwing errors.

Reply:
The node will also print the ‘uncaught rejections’ when there’s any fatal error.

however, I can agree with you of your points.

Reply:
Has been re-deleted.